### ISSUE TITLE
Layer support files for mingw-w64-vulkan-validation-layers

### ISSUE THREAD

**Issue by xzn** [msys2/MINGW-packages#6799]
Trying to build https://github.com/aejsmith/vkdevicechooser but vk`vulkan/vk_layer_dispatch_table.h` is missing.

This could be worked around by changing [`PKGBUILD`](https://github.com/msys2/MINGW-packages/blob/feeb9c28b064d4e587498f0140424c5253076b8d/mingw-w64-vulkan-validation-layers/PKGBUILD)

```diff
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
     -DGLSLANG_INSTALL_DIR=${MINGW_PREFIX} \
     -DSPIRV_HEADERS_INSTALL_DIR=${MINGW_PREFIX} \
-    -DBUILD_LAYER_SUPPORT_FILES=OFF \
+    -DBUILD_LAYER_SUPPORT_FILES=ON \
+    -DCMAKE_INSTALL_INCLUDEDIR=include/vulkan \
     -DBUILD_TESTS=OFF \
     ../${_realname}-${pkgver}
```

The resulting layer seems to compile and work.

**Comment by Biswa96** [comment]
Would you like to create a pull request to add the change? Contributions are always welcomed 🤗

**Comment by lazka** [comment]
I guess this is fixed by #6800
